Market/Controllers/LoginController.cs

```diff
@@ -33,15 +33,6 @@ public async Task<IActionResult> Register([FromBody] RegisterDto registerDto)
         {
```

Comment:
You need to add validation here: `if (!ModelState.IsValid) return BadRequest(ModelState);`